app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt

```diff
@@ -27,18 +27,18 @@ import com.google.samples.apps.sunflower.data.AppDatabase
 import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
-class SeedDatabaseWorker(val context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
+class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
```

Comment:
`...:Worker(context.applicationContext, ...`, because,  ```SeedDatabaseWorker(context: Context,...``` might be called by an  `Activity` .

Reply:
@tiembo Please consider the PR #97 which has solved the similar issue there.

Reply:
This is the preferred approach for Workers

Reply:
@tiembo 
Take a look at here
```
public abstract class NonBlockingWorker {

    @SuppressWarnings("NullableProblems")   // Set by internalInit
    private @NonNull Context mAppContext;
```
The grandfather of `Worker` has a member of  `Context`.  If `SeedDatabaseWorker` takes an `Activity` as `context` and passes to `Worker`, there could be a  `leak`.